src/components/organisms/browse/Map.jsx

```diff
@@ -59,12 +60,19 @@ function UpdateCenter({ fridgeList }) {
   });
 }
 
-export default function Map({ fridgeList }) {
+function FocusMarkerOnMap({ marker }) {
```

Comment:
This doesn't seem like it should be it's own component. Is there a reason why this is necessary vs moving this logic into a useCallback wrapped function inside of the map component?

Reply:
yeah the function uses useMap hook and it requires to be inside of MapContainer, since useMap looks for a context provided by MapContainer

Reply:
and I also liked the fact that it can be used like a component, that kinda make things easy to look at